### ISSUE TITLE
dynamic viscosity of propylene glycol ?

### ISSUE THREAD

**Issue by thermengineer** [CoolProp/CoolProp#918]
Dear colleagues
It seems that coolprop turns dynamic viscosity of Ashrae propylene glycol (APG) ten times lower than expected. Example: 
Fluid:ASHRAE, Propylene Glycol (APG)
Volumetric Brine Ratio:40.00 %
Temperature: 35.00 °C
Atmospheric Pressure: 101325 Pa

COOLPROP Dynamic Viskosity: 0.26387 mPa-s
ASHRAE Dynamic Viskosity*: 2.6 mPa-s

(*Ref: 2005 ASHRAE Handbook—Fundamentals (SI) CHAPTER 21: (PHYSICAL PROPERTIES OF SECONDARY COOLANTS (BRINES)) - Page 21.9 / Table 13 Viscosity of Aqueous Solutions of Propylene Glycol)

Could you check if I miss something?

Thank you,

**Comment by jowr** [comment]
Which version are you using? I fixed some viscosities in December. My version has the correct values
```Python 
In [1]: import CoolProp
In [2]: from CoolProp.CoolProp import PropsSI
In [3]: print CoolProp.__version__
5.1.3dev
In [4]: print CoolProp.__gitrevision__
468ed784e8035def036eacad927998b8682831e3
In [5]: print PropsSI('V','T',35+273.15,'P',10e5,'INCOMP::APG-40%')*1e3
2.63873017128
```

**Comment by thermengineer** [comment]
Dear Wronski, Thank you for your answer. I use previous version. Let me check and inform you.

**Comment by jowr** [comment]
The changes should be included in v5.1.2, you do not need 5.1.3dev.

**Comment by ibell** [comment]
bump.  Does this fix it?
